Tests/BootCamp.Chapter.Tests/DeckTest.cs

```diff
@@ -12,15 +13,21 @@ public class DeckTest
         [Fact]
         public void New_Given_Null_List_Throws_NullArgumentException()
```

Comment:
string, int, list- we don't really get a contextual meaning from the type. Please give them logical meaning. In this case it's just wrong on many levels.

Reply:
In this case it's supposed to be cards and not a deck.